configd/src/apps/sentinel/env.cpp

```diff
@@ -38,6 +41,23 @@ void Env::boot(const std::string &configId) {
         configId.c_str(), cfg.port.telnet, cfg.port.rpc);
     rpcPort(cfg.port.rpc);
     statePort(cfg.port.telnet);
+    if (auto up = ConfigOwner::fetchModelConfig(MODEL_TIMEOUT_MS)) {
+        const ModelConfig &model = *up;
+        for (const auto & h : model.hosts) {
+            LOG(info, "- Model for host %s with %zd services", h.name.c_str(), h.services.size());
+            for (const auto & s : h.services) {       
+                if (s.name == "config-sentinel") { 
+                    LOG(info, "  - Model for service %s type %s configid %s with %zd ports",
+                        s.name.c_str(), s.type.c_str(), s.configid.c_str(), s.ports.size());
+                    for (const auto & p : s.ports) {
+                        if (p.tags.find("rpc") != p.tags.npos) {
+                            LOG(info, "    - Model for port %d has tags %s", p.number, p.tags.c_str());
```

Comment:
will spamming the log have any negative impact on production?

Reply:
considering the logging from other things that happen at startup (just metricsproxy logs > 40 lines) I'm not worried.  But this logging will be removed when we actually use the model config later, I guess - it's mostly a placeholder.